cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+
+# OpenID Connect claim handlers
+OAUTH_OIDC_ID_TOKEN_HANDLERS = (
+    'oauth2_provider.oidc.handlers.BasicIDTokenHandler',
+    'oauth2_provider.oidc.handlers.ProfileHandler',
+    'oauth2_provider.oidc.handlers.EmailHandler',
+    # Includes the "administrator" claim in the token. This currently resides in lms/djangoapps!
+    'oauth2_handler.IDTokenHandler'
```

Comment:
edx-oauth2-provider intentionally makes it difficult to [customize scopes](https://github.com/edx/edx-oauth2-provider/blob/master/oauth2_provider/constants.py#L65-L69). Given the circumstances, it looks like creating these JWTs manually is justifiable. Doing so will result in limited code duplication, but it'll let us get the claims we need while staying away from most of the OAuth2 code, meaning it'll be easier to understand what's going on.